app/models/loan.rb

```diff
@@ -237,8 +237,12 @@ def thumb_path
     else "/assets/ww-avatar-watermark.png" end
   end
 
+  def ensure_currency
+    currency || Currency.find_by(code: 'USD')
+  end
```

Comment:
You're right. It looks like currencies aren't being properly migrated. I'll look into that.